x-pack/plugins/enterprise_search/public/applications/workplace_search/components/overview/onboarding_card.test.tsx

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import '../../../__mocks__/shallow_usecontext.mock';
+
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+
+import { EuiFlexItem, EuiButton, EuiButtonEmpty } from '@elastic/eui';
+
+import { OnboardingCard } from './onboarding_card';
+
+jest.mock('../../../shared/telemetry', () => ({ sendTelemetry: jest.fn() }));
+import { sendTelemetry } from '../../../shared/telemetry';
+
+const cardProps = {
+  title: 'My card',
+  icon: 'icon',
+  description: 'this is a card',
+  actionTitle: 'action',
+  testSubj: 'actionButton',
+  actionPath: '/foo_path',
+};
+
+describe('OnboardingCard', () => {
+  it('renders', () => {
+    const wrapper = shallow(<OnboardingCard {...cardProps} />);
+    expect(wrapper.find(EuiFlexItem)).toHaveLength(1);
+  });
+
+  it('renders an action button', () => {
+    const wrapper = mount(<OnboardingCard {...cardProps} />);
+    const button = wrapper.find('a[data-test-subj="actionButton"]');
```

Comment:
I think we can simplify this by `dive()`ing into EuiEmptyWrapper, instead of using `mount()`. At times it's more of a pain to shallow+dive than to just mount, but in this one scenario it's not too bad at all. 

Example:
```suggestion
    const wrapper = shallow(<OnboardingCard {...cardProps} />);
    const prompt = wrapper.find(EuiEmptyPrompt).dive();
    const button = prompt.find('[data-test-subj="actionButton"]');
```

Reply:
Addressed in 2bedde2